examples/tv-app/android/java/AppPlatform-JNI.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <jni.h>
+
+class AppPlatformJNI
```

Comment:
why is this a class instead of a namespace?

with a namespace you could have a public GetInstance() and in a cpp file you can have the static sInstance.

Reply:
Unused code is removed and documentation added for all the doc callouts here in a PR following this one. https://github.com/project-chip/connectedhomeip/pull/22354